Speech/src/V1/resources/speech_rest_client_config.php

```diff
@@ -35,19 +29,6 @@
             'ListOperations' => [
                 'method' => 'get',
                 'uriTemplate' => '/v1/operations',
-                'additionalBindings' => [
```

Comment:
Should we be worried about this getting removed?

Reply:
`additionalBindings` are to allow multiple urlTemplates for a single RPC. However, as this layer is handled by `LongRunning\OperationsClient`, and that client will work with the remaining bindings, there is no problem here. These endpoints will result (and have been resulting) in a 404 anyway.